state-sync/src/fuzzing.rs

```diff
@@ -38,16 +35,14 @@ proptest! {
     }
 }
 
-pub fn test_state_sync_msg_fuzzer_impl(msg: StateSyncMessage) {
+pub fn test_state_sync_msg_fuzzer_impl(message: StateSyncMessage) {
     block_on(async move {
-        STATE_SYNC_COORDINATOR
+        let _ = STATE_SYNC_COORDINATOR
             .lock()
-            .process_one_message(
-                PeerNetworkId(
-                    NodeNetworkId::new(NetworkId::Validator, 0),
-                    PeerId::new([0u8; PeerId::LENGTH]),
-                ),
-                msg,
+            .process_chunk_message(
+                NodeNetworkId::new(NetworkId::Validator, 0),
+                PeerId::new([0u8; PeerId::LENGTH]),
+                message,
             )
             .await;
```

Comment:
`unwrap()`?  I'm assuming you want to determine the output of the function?

Reply:
No, in this case we're running the fuzzer which may cause an error or an Ok(()). We don't care, we only care if the fuzzer crashes the code 😄 (which is caught by an external handler)

Reply:
Okay, so we're only looking for panics, which wouldn't be handled by the Result.